Training.Core/Models/Book.cs

```diff
@@ -13,6 +13,7 @@ public class Book : IRemovable
         public string Name { get; set; }
         public string Author { get; set; }
         public bool IsDeleted { get; set; }
+        public Album Album { get; set; }
```

Comment:
Perhaps we are having a different concept of "Album", I see like a music album, and being something completly isolated, I think the relation with Book is not necessary

However, despite that for me functionally it has no sense, technically speaking, the relation with EFCore has been correctly done ;)